doc/release-notes.md

```diff
@@ -223,6 +223,15 @@ modules were reorganized in separate folders to make navigation through code a b
 
 See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.14.0.x...dashpay:v0.15).
 
+- [`1d9adbe639`](https://github.com/dashpay/dash/commit/1d9adbe639) Replace generic CScopedDBTransaction with specialized CEvoDBScopedCommitter (#3292)
+- [`8fd486c6bc`](https://github.com/dashpay/dash/commit/8fd486c6bc) Translations 2020-01 (#3192)
+- [`3c54f65278`](https://github.com/dashpay/dash/commit/3c54f65278) Bump copyright year to 2020 (#3290)
+- [`35d28c7485`](https://github.com/dashpay/dash/commit/35d28c7485) Update man pages (#3291)
+- [`203cc90778`](https://github.com/dashpay/dash/commit/203cc90778)  trivial: adding SVG and high contrast icons  (#3209)
+- [`e875d4925a`](https://github.com/dashpay/dash/commit/e875d4925a) Define defaultTheme and darkThemePrefix as constants and use them instead of plain strings (#3288)
+- [`1d203b422c`](https://github.com/dashpay/dash/commit/1d203b422c) Bump PROTOCOL_VERSION to 70216 (#3287)
+- [`b84482ac57`](https://github.com/dashpay/dash/commit/b84482ac57) Let regtest have its own qt settings (#3286)
+- [`1c885bbedf`](https://github.com/dashpay/dash/commit/1c885bbedf) Only load valid themes, fallback to "Light" theme otherwise (#3285)
```

Comment:
nit: not sure why, but my short commit hashes produced via `git log --oneline --no-merges master..HEAD` were slightly shorter...

Reply:
4ff5cdebb

Reply:
Yeah noticed this as well, my assumption is that I have a local conflict when shorter hashes are used

Reply:
@UdjinM6 your commit only touched the part in [] but not the link itself. Is this what you wanted?

Reply:
ooops 🙈 187250863

Reply:
would probably make sense to squash all fixes into one `Update release-notes.md` commit

Reply:
done